index.md

```diff
@@ -1444,6 +1444,28 @@ Numeric Replies:
     <p>NOTE: The <tt>KILL</tt> message is weird, and I need to look at it more closely, add some examples, etc.</p>
 </div>
 
+### SQUIT message
+
+         Command: SQUIT
+      Parameters: <server> <comment>
+
+The SQUIT command is available only to operators.  It is used to disconnect server links.
+
+The `<comment>` SHOULD be supplied by all operators who execute a SQUIT for a remote server.
```

Comment:
Simplifies the first paragraph, and mentions how this command acts in a traditional spanning-tree topology. I know that how servers act in S2S isn't something we wanna specify or mention, but here it makes sense since this is a command that deals with S2S links.

Insp has mentioned that they wanna do this for v4 (rather than their current [`SQUIT` vs `RSQUIT` split](https://docs.inspircd.org/3/modules/spanningtree/#commands)) so we should be fine writing this here.

```suggestion
The `SQUIT` command disconnects a server from the network. `SQUIT` is a privileged command and is only available to IRC Operators. `<comment>` is the reason why the server link is being disconnected.

In a traditional spanning-tree topology, the command gets forwarded to the specified server. And the link between the specified server and the last server to propagate the command gets broken.
```